modules/ROOT/pages/cryptography-xml.adoc

```diff
@@ -102,6 +102,38 @@ In the next example, the asymmetric key is used to validate the signature of the
 <crypto:xml-validate config-ref="asymmetricConfig" keyId="myAsymKey" elementPath="/PurchaseOrder/Buyer"/>
 ----
 
+If the document to validate has multiple signatures, the `elementPath` attribute must be set and has to select any of the signatures. The XPath expression specifies the element that has the signature to validate.
```

Comment:
It's more of a relationship of being the target of the signature, more than having the signature itself.